### ISSUE TITLE
Issues with Examples

### ISSUE THREAD

**Issue by bruce987** [takuya-takeuchi/OpenPoseDotNet#13]
Hi takuya-takeuchi,

I've been messing with this for a few weeks now. Today I started fresh and used your original steps to build it. One of the problems I see is in the steps "Try Tutorial". The first one "01_BodyFromImageDefault" shows these steps...
> cd <OpenPoseDotNet_dir>\examples\TutorialApiCpp\01_BodyFromImageDefault
> **SymlinkBinary.bat <Debug/Release> build_win_desktop_<cpu/cuda>_x64**
> dotnet run -c Release  -i "examples\media\COCO_val2014_000000000192.jpg" 

All of the others show the following steps...
> cd <OpenPoseDotNet_dir>\examples\TutorialApiCpp\08_HeatmapsFromImage
> **SymlinkBinary.bat <Debug/Release>**
> dotnet run -c Release  -i "examples\media\COCO_val2014_000000000294.jpg"

Noticed that the SymlinkBinary.bat file leaves out the second argument. After fixing the error in execution, I tried running some of the examples. The first one works! "**01_BodyFromImageDefault**"
All of the other examples generate this error...
![image](https://user-images.githubusercontent.com/1581231/78410991-c57bde00-75d3-11ea-8874-b500ab0a7bae.png)

**Comment by takuya-takeuchi** [comment]
@bruce987 
Yeah, you are correct. README.md and bat file explanation have fucking wrong.
OK, let me fix it.

Thanks.

**Comment by takuya-takeuchi** [comment]
### NOTE

* README.md
* SymlinkBinary.bat
  * Add check whether argument is passed or not
  * Fix link to examples and models directory

**Comment by takuya-takeuchi** [comment]
Corrected document and script.
Please get latest code from develop branch.
Replace SymlinkBinary.bat with ymlinkBinary.ps1.

Just in case, please delete bin, examples, models and obj from each example directory before run SymlinkBinary.ps1.
Because wrong  symlink path occurs issue you faced to.

**Comment by bruce987** [comment]
@takuya-takeuchi 

I pulled the development branch. Cleanup the folders of bin, obj, models, and examples. I ran powershell as administrator and got the following error:
![image](https://user-images.githubusercontent.com/1581231/79053102-5d934c00-7c00-11ea-9e3d-f35f3601e99c.png)

**Comment by takuya-takeuchi** [comment]
@bruce987 
Your machine does not allow you to execute untrusted scripts.
Please check and follow the this page.
https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.security/set-executionpolicy?view=powershell-7

**Comment by bruce987** [comment]
Ok I’ll do it tomorrow... it’s late here in Texas and I’ve had a few beers!

**Comment by bruce987** [comment]
I changed the security policy and it now runs, but doesn't find one of the boost libraries:
![image](https://user-images.githubusercontent.com/1581231/79077064-1a9aac80-7cc4-11ea-80da-c6a1cf354e5d.png)

...and here is the build folder
![image](https://user-images.githubusercontent.com/1581231/79077088-5170c280-7cc4-11ea-948c-556a521d5078.png)

**Comment by takuya-takeuchi** [comment]
Ok, I understand.
I used VS 2015 and it created vc140.dll.
Sorry, I forgot to consider version of VS.

**Comment by bruce987** [comment]
I have you considered moving the SymlinkBinary.ps1 up two folders, "examples". Then when it is run it applies all the necessary data setup for all examples. This way you don't need to maintain multiple script files.

**Comment by bruce987** [comment]
Hey Takuya,

 

I did some research on the OpenPose application (not your DotNet Library). My laptop now has VS 2015, 2017 and 2019 installed on it. It has Nvidia RTX 2080 graphics card. I cloned openpose and built it using their recommended approach which is cmake-gui. I also have CUDA 10.0, 10.1, and 10.2. CUDA 10.0 was the initial version referenced. 

 

I cloned openpose and built it using VS2015. I verified that it worked by running their demo.

 

Next I cloned openpose into a folder called openpose2017. Built it using VS2017. I verified that it worked by running their demo.

 

Finally, I cloned openpose into a folder called openpose2019. Tried building it using VS2019. At this point the build failed due to the CUDA version: “include\crt\host_config.h”. Apparently CUDA 10.0 does NOT work with VS2019. I checked the “include\crt\host_config.h” file for 10.1 and 10.2. Both those files showed valid versions for VS2019. I switched to CUDA 10.2 and cleaned up everything – deleted the cloned folder. I started over. It built and I verified that it worked by running the demo.

 

In conclusion, if you support VS2019, you will need to make checks for CUDA 10.1 or greater when building openpose.

 

Thanks,

Bruce

**Comment by bruce987** [comment]
@takuya-takeuchi 

I modified the SymlinkBinary.ps1... I removed the dll's that it couldn't find. That fixed the problem.

-Bruce
